tests/test_api.py

```diff
@@ -1,63 +1,63 @@
-    assert response.reason_phrase == "OK"
-
-
-def test_post_byte_iterator(server):
-    def data():
-        yield b"Hello"
-        yield b", "
-        yield b"world!"
-
-    response = httpx.post(server.url, data=data())
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_options(server):
-    response = httpx.options(server.url)
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_head(server):
-    response = httpx.head(server.url)
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_put(server):
-    response = httpx.put(server.url, data=b"Hello, world!")
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_patch(server):
-    response = httpx.patch(server.url, data=b"Hello, world!")
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_delete(server):
-    response = httpx.delete(server.url)
-    assert response.status_code == 200
-    assert response.reason_phrase == "OK"
-
-
-def test_get_invalid_url(server):
-    with pytest.raises(httpx.InvalidURL):
-        httpx.get("invalid://example.org")
+# import pytest
+#
+# import httpx
+#
+#
+# def test_get(server):
+#     response = httpx.get(server.url)
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#     assert response.text == "Hello, world!"
+#     assert response.http_version == "HTTP/1.1"
+#
+#
+# def test_post(server):
+#     response = httpx.post(server.url, data=b"Hello, world!")
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_post_byte_iterator(server):
+#     def data():
+#         yield b"Hello"
+#         yield b", "
+#         yield b"world!"
+#
+#     response = httpx.post(server.url, data=data())
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_options(server):
+#     response = httpx.options(server.url)
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_head(server):
+#     response = httpx.head(server.url)
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_put(server):
+#     response = httpx.put(server.url, data=b"Hello, world!")
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_patch(server):
+#     response = httpx.patch(server.url, data=b"Hello, world!")
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_delete(server):
+#     response = httpx.delete(server.url)
+#     assert response.status_code == 200
+#     assert response.reason_phrase == "OK"
+#
+#
+# def test_get_invalid_url(server):
+#     with pytest.raises(httpx.InvalidURL):
+#         httpx.get("invalid://example.org")
```

Comment:
Commented out until we flesh out the async high-level API, right? :-)

Reply:
Included top-level async API. We'll probably want to switch the top-level stuff to a global client/connection pool rather than a one-shot, but it's still valuable in the meantime.